website/templates/util/render_node.mako

```diff
@@ -102,33 +102,37 @@
         % if not summary['archiving']:
         <div class="body hide" id="body-${summary['id']}" style="overflow:hidden;">
             <hr />
-            Recent Activity
-            <!-- ko stopBinding: true -->
-            <div id="logs-${summary['id']}" class="log-container" data-uri="${summary['api_url']}log/">
-                <dl class="dl-horizontal activity-log" data-bind="foreach: {data: logs, as: 'log'}">
-                    <dt><span class="date log-date" data-bind="text: log.date.local, tooltip: {title: log.date.utc}"></span></dt>
-                    <dd class="log-content">
-                        <span data-bind="if:log.anonymous">
-                           <span data-bind="html: $parent.anonymousUserName"></span>
-                        </span>
-                        <span data-bind="ifnot:log.anonymous">
-                            <a data-bind="text: log.userFullName || log.apiKey, attr: {href: log.userURL}"></a>
-                        </span>
+            % if summary['is_retracted']:
+                <h4>Recent activity information has been retracted.</h4>
+            % else:
+                Recent Activity
+                <!-- ko stopBinding: true -->
+                <div id="logs-${summary['id']}" class="log-container" data-uri="${summary['api_url']}log/">
+                    <dl class="dl-horizontal activity-log" data-bind="foreach: {data: logs, as: 'log'}">
+                        <dt><span class="date log-date" data-bind="text: log.date.local, tooltip: {title: log.date.utc}"></span></dt>
+                        <dd class="log-content">
+                            <span data-bind="if:log.anonymous">
+                               <span data-bind="html: $parent.anonymousUserName"></span>
```

Comment:
Going forward, I think we should strongly discourage allowing the ViewModel to write arbitrary HTML this way. (*["When using KO, your view models are pure JavaScript objects that hold no knowledge of HTML"](http://knockoutjs.com/documentation/observables.html#mvvm_and_view_models)*).

Move the `<em>` tags from `logFeed.js` to the template. You may be able to get rid of one or both `span` tags in the process, simplifying the markup.